Umweltzone/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,327 @@
+
+    <!-- AboutActivity -->
+    <string name="appinfo_application_logo_description">
+        Logo da aplicação &#8220;Umweltzone&#8221;
+    </string>
+    <string name="about_activity_title">Sobre</string>
+    <string name="appinfo_subtitle">
+        Um app Android para zonas de baixa emissão (Em Alemão: Umweltzonen) e zonas de proibição de Diesel na Alemanha
+    </string>
+
+    <!-- AboutActivity / Copyright -->
+    <string name="appinfo_copyright_title">Ideação e concretização</string>
+    <string name="appinfo_copyright_content">
+        (&#169;) Tobias Preuss, 2013 &#8211; 2020
+    </string>
+
+    <!-- AboutActivity / Description -->
+    <string name="appinfo_description_title">Descrição</string>
+
+    <string name="appinfo_description_content">
+        Este aplicativo mostra a localização de zonas de baixa emissão (ZBE),
+        zonas de proibição de Diesel (ZPD) e fornece informações sobre ambos os assuntos.
+        A entrada em um ZBE é restrita a veículos marcados com um adesivo específico.
+        O adesivo corresponde ao nível de emissão do veículo. Os veículos que não atendem
+        a uma determinada norma de emissão de gases de Diesel estão restritos a entrar em uma ZPD.
+        O objetivo das restrições de entrada é o de aumentar a qualidade do ar nas cidades.
+        As partículas finas devem ser reduzidas para preservar a saúde do cidadão. &#8212;
+        Além do mapa e da lista de cidades, há uma tela detalhada que contém informações individuais
+        da cidade selecionada. As perguntas mais frequentes podem ser encontradas no item &#8220;FAQ&#8221; do menu.
+        Além disso, existem links para sites que fornecem mais informações sobre ZBE, ZPD e mais assuntos relacionados.
+    </string>
+
+
+    <!-- AboutActivity / Disclaimer -->
+    <string name="appinfo_disclaimer_title">Dados, qualidade e atualizações</string>
+    <string name="appinfo_disclaimer_content">
+        Os dados e as coordenadas das zonas disponíveis em formato digital foram adquiridos diretamente das cidades ou governos.
+        O OpenStreetMap também foi uma fonte importante, uma vez que várias zonas de baixa emissão puderam ser encontradas por lá.
+        Além deles, algumas zonas foram digitalizadas manualmente com base nas publicações oficiais da Agência Ambiental Federal.
+        Devido ao fato de que muitas autoridades publicam gráficos de baixa resolução de mapas digitalizados, não se tem a garantia
+        de que as fronteiras da zona correspondem precisamente à realidade. Os dados e coordenadas da zona variam de acordo com sua fonte,
+        qualidade e precisão no processo de digitalização. Além disso, não há uma automação na informação sobre as alterações feitas
+        pelas cidades. Os dados da zona são atualizados manualmente e dependem muito do feedback de usuários amigáveis ​​como você.
+        Por estes motivos não garantimos uma visualização perfeita e não nos responsabilizamos por custos ou punições.
```

Comment:
There are strange characters in the file at `... usuários amigáveis ​​como você ...` - it looks like this is a typo to be removed.

Reply:
It actually doesn't have any typos in Portuguese, I'm gonna replace it by some words without signs and with the same meaning so we can see if any error is avoided.

By the way, I wasn't able to run all the parts of the app. I believe I can't be in Brazil in order to do that.

Reply:
When I was about to commit I realized that was a weird character that was caught git diff command. So I just deleted and turn it back to the original version.

Reply:
It works without a crash now. Thanks.
Regarding starting the app you have to switch the system language of your device to Portuguese. Then the app show show up in Portuguese instead of English.